transform.go

```diff
@@ -49,6 +51,14 @@ func Transform(img []byte, opt Options) ([]byte, error) {
 		return nil, err
 	}
 
+	if (opt.BackgroundColor != color.RGBA{} && format == "png" || format == "tiff") && opt.Format == "jpeg" {
```

Comment:
Maybe some more parens would help here. It's hard to see at a glance what this if statement is checking.
I think `(opt.BackgroundColor != color.RGBA{} && format == "png" || format == "tiff")` could be interpreted either as `A && (B || C)` or `(A && B) || C`

Reply:
Makes sense. I've cleared it up.